retrofit/src/main/java/retrofit/RequestBuilder.java

```diff
@@ -381,6 +379,12 @@ Request build() {
     }
 
     RequestBody body = this.body;
+    if (formEncodingBuilder != null) {
```

Comment:
Smells a bit weird that this is an `if`/`else` block, as if form encoding has higher priority than multipart, when in reality they're mutually exclusive.